apps/sumo/infra/shared/main.tf

```diff
@@ -75,3 +80,35 @@ module "sumo-user-media-prod-cf" {
     domain_name = "sumo-user-media-prod.s3-website-us-west-2.amazonaws.com"
 }
 
+#####################################################################
+# static media CDN
+#####################################################################
+module "sumo-static-media-dev-cf" {
+    source = "./cloudfront_static_media"
+
+    acm_cert_arn = "TODO"
+    aliases = ["static-media-dev-cdn.sumo.mozilla.net", "static-media-dev-cdn.sumo.moz.works"]
+    comment = "Dev CDN for SUMO static media"
+    distribution_name = "SUMOStaticMediaDevCDN"
+    domain_name = "TODO"
```

Comment:
sumo-dev.frankfurt.moz.works

Reply:
- dev.sumo.moz.works CNAME -> ELN (https) -> sumo-dev.frankfurt.moz.works
- we will need a valid cert